fhir-registry/src/main/java/com/ibm/fhir/registry/FHIRRegistry.java

```diff
@@ -326,6 +350,15 @@ private Resource getResource(FHIRRegistryResource registryResource, String url,
         return providers;
     }
 
+    /**
+     * initializes the Resource Providers.
+     */
+    public static void init() {
+        for (FHIRRegistryResourceProvider provider : getInstance().providers) {
+            provider.init();
```

Comment:
most providers are loaded from `loadProviders` during static intialization.  
however, we also support adding providers at runtime via `addProvider`...I think we should throw a `provider.init()` call into that method as well.

Reply:
excellent point. I'll do so

Reply:
added